sourcegraph/learn
# Remove globby

### What should this PR do?
Part of the work on [DEVED-93](https://sourcegraph.atlassian.net/browse/DEVED-93), this ticket ([DEVED-111](https://sourcegraph.atlassian.net/browse/DEVED-111)) removes the `globby` dependency in favor of using `fs`.

As of Next.js (9.4+), [we get `fs` out of the box](https://github.com/vercel/next.js/issues/7755#issuecomment-508633125). 

This PR also removes our empty `guides` directory, and the `guides/[slug].tsx` template. See the discussion below for more context on this decision.

### Why are we making this change?
- We are working on updating dependencies, and removing anything we don't need will lead to fewer dependencies and breakages on upgrade in the future.

### What are the acceptance criteria? 
- App should work as expected, all files should load as expected.

### How should this PR be tested?
- Check out the branch, remove `node_modules`, and install dependencies with `npm ci`.
- Check that files load properly and things work as expected.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
